metricbeat/module/rabbitmq/connection/connection.go

```diff
@@ -0,0 +1,58 @@
+package node
+
+import (
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/common/cfgwarn"
+	"github.com/elastic/beats/metricbeat/helper"
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/elastic/beats/metricbeat/mb/parse"
+)
+
+const (
+	defaultScheme = "http"
+	defaultPath   = "/api/connections"
+)
+
+var (
+	hostParser = parse.URLHostParserBuilder{
+		DefaultScheme: defaultScheme,
+		DefaultPath:   defaultPath,
+	}.Build()
+)
+
+func init() {
+	if err := mb.Registry.AddMetricSet("rabbitmq", "connection", New, hostParser); err != nil {
+		panic(err)
+	}
+}
+
+type MetricSet struct {
+	mb.BaseMetricSet
+	*helper.HTTP
+}
+
+func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
+	cfgwarn.Experimental("The rabbitmq connection metricset is experimental")
```

Comment:
Somehow docs and code do not align here. In the docs it's marked as beta. I assume that has to do with our generator :-( Could you change this to `cfgwarn.Beta`  here?